tests/_internal/mbed_tools/test_export.py

```diff
@@ -29,3 +29,21 @@ def test_export(self, mock_write_cmakelists_file, mock_generate_cmakelists_file)
         mock_write_cmakelists_file.assert_called_once_with(
             pathlib.Path(output_dir), mock_generate_cmakelists_file.return_value
         )
+
+    @mock.patch("mbed_build._internal.mbed_tools.export.generate_cmakelists_file")
+    @mock.patch("mbed_build._internal.mbed_tools.export.write_cmakelists_file")
+    def test_export_default_project_path(self, mock_write_cmakelists_file, mock_generate_cmakelists_file):
+        mock_file_contents = "Hello world"
+        mock_generate_cmakelists_file.return_value = mock_file_contents
+        output_dir = "some-directory"
+        mbed_target = "K64F"
+        toolchain = "GCC"
+
+        runner = CliRunner()
+        result = runner.invoke(export, ["-o", output_dir, "-t", toolchain, "-m", mbed_target])
+
+        self.assertEqual(result.exit_code, 0)
+        mock_generate_cmakelists_file.assert_called_once_with(mbed_target, ".", toolchain)
```

Comment:
I'm never sure if it's better to resolve `.` at read time or not. I.e.: `.` right now might mean something else when it's used down the line, in case something changes working directory.